specification/monitor/resource-manager/microsoft.insights/preview/2017-09-01-preview/metricAlert_API.json

```diff
@@ -0,0 +1,888 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request for a list of metric alerts",
+            "schema": {
+              "$ref": "#/definitions/MetricAlertResourceCollection"
+            },
+            "examples": {
+              "application/json": {
+                "value": [
+                  {
+                    "location": "global",
+                    "tags": {
+                      "key1": "value1",
+                      "key2": "value2"
+                    },
+                    "id": "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourcegroups/test/providers/microsoft.insights/metricalerts/storageTest",
+                    "name": "storageTest",
+                    "properties": {
+                      "description": "This is the description of the xstore multidim rule1",
+                      "severity": 3,
+                      "enabled": true,
+                      "scopes": [
+                        "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/test/providers/Microsoft.Storage/storageAccounts/testmultidim/blobServices/default"
+                      ],
+                      "evaluationFrequency": "PT1M",
+                      "windowSize": "PT15M",
+                      "criteriaSchemaId": "SingleResourceMultipleMetricCriteria",
+                      "criteria": {
+                        "allOf": [
+                          {
+                            "name": "HowManyBytes",
+                            "metricName": "BlobCapacity",
+                            "dimensions": [
+                              {
+                                "name": "BlobType",
+                                "operator": "Include",
+                                "values": [
+                                  "BlockBlob",
+                                  "PageBlob"
+                                ]
+                              }
+                            ],
+                            "operator": "GreaterThanOrEqual",
+                            "threshold": 5.0,
+                            "timeAggregation": "Maximum"
+                          }
+                        ]
+                      },
+                      "actions": [
+                        {
+                          "actionGroupId": "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/test/providers/microsoft.insights/actionGroups/testActionGroup"
+                        }
+                      ],
+                      "currentStatus": {
+                        "value": "Healthy",
+                        "timestamp": "2018-01-17T20:17:25.6293237Z"
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": { "$ref": "#/definitions/CloudError" }
```

Comment:
Make sure this name is the same we use in the other specs. So that our customers know what type will be returned in an error and we do not need to change PowerShell later.

Reply:
@LeoVannelli if the type is defined in another spec, I'd actually recommend referencing it directly (see [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/networkInterface.json#L213) for example syntax).